README.md

```diff
@@ -4,142 +4,31 @@ This repo is a monorepo, which contains the pieces that make up the Universal Co
 application which anyone can clone and self-host as a way to serve the connect widget via a url which can then be loaded
 into an iframe.
 
-## Documentation
-
-Please refer to the [UCP documentation](https://docs.universalconnect.org) (Coming soon) for additional information on how to use the widget.
-
-## Authentication
-
-The express endpoints exposed in these repositories don't provide any authentication. You will need to fork the repo if you want to add your own authentication.
-
-## Api documentation
-
-The api documentation for this service lives in openApiDocumentation.json and uses the (open api spec)[https://swagger.io/specification/]. You can open it in your preferred tool. You may copy the file into [swagger editor](https://editor.swagger.io/) and edit the local file with updates when complete.
-
-## Cached data
-
-This widget runs off of cached data, which is included in this repo, so that it doesn't rely on any UCP-hosted services in order to function.
-
-In `./apps/server/cachedDefaults` you will find files that are loaded into a redis cache. In the future these files will be used as a backup in case UCP-hosted services are down.
-
-The following files need to be there for the widget service to function
-
-1. Preferences
-1. Institutions
-
-## Getting Started (production)
+## Table of Contents
 
-To get started: clone the repo, follow the steps in [Getting Started](#getting-started-in-development) and
-[Initial Setup](#initial-setup) to set up your `.env` files, and then run the following command from the root of the 
-project:
+- [Getting Started](#getting-started)
```

Comment:
This table of contents seems out of date now

Reply:
Yep, thanks.